src/BlockTypes/MiniCart.php

```diff
@@ -363,7 +363,6 @@ protected function get_markup( $attributes ) {
 		$cart_controller     = $this->get_cart_controller();
 		$cart                = $cart_controller->get_cart_instance();
 		$cart_contents_count = $cart->get_cart_contents_count();
-		$cart_contents       = $cart->get_cart();
```

Comment:
Changes in this file are generic cleanup. Unrelated to the other changes in this PR.